Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml

```diff
@@ -89,6 +89,10 @@
     ClothingShoesBootsCowboyBrown: 1
     ClothingShoesBootsCowboyBlack: 1
     ClothingShoesBootsCowboyWhite: 1
+  contrabandInventory: #Delta V: Added Techpriest Robe to Contra Inventory
```

Comment:
Wait why is it in the autodrobe's contra inventory now :imcry:

Reply:
I mean I was gonna make people work for em a little bit but i can zap that out then :P